app/src/main/res/values/strings.xml

```diff
@@ -174,20 +174,20 @@
     <!-- error dialogs -->
     <string name="dialog_error_details_button_default">Show details</string>
     <string name="dialog_error_details_default">No details available</string>
-    <string name="dialog_error_details_write">This station is probably a duplicate or else you might want to check your external storage.</string>
+    <string name="dialog_error_details_write">Check your external storage if this station isn't a duplicate.</string>
     <string name="dialog_error_title_fetch_download">Download Issue</string>
     <string name="dialog_error_title_fetch_read">Read Issue</string>
     <string name="dialog_error_title_fetch_write">Write Issue</string>
     <string name="dialog_error_message_fetch_general_external_storage">External storage:</string>
-    <string name="dialog_error_message_fetch_general_hint_m3u">Hint: Transistor can only process MP3, AAC and Ogg/Opus encoded streams or links ending in .m3u or .pls.</string>
+    <string name="dialog_error_message_fetch_general_hint_m3u">Hint: Transistor can only process MP3, AAC and Ogg Opus encoded streams or links ending in .m3u or .pls.</string>
```

Comment:
As per https://en.wikipedia.org/wiki/Opus_%28audio_format%29

Reply:
As I understand the Wikipedia entry the actual data format of such a stream is OPUS and OGG is just a container. If that is the case we could drop "Ogg" completely, right?

Reply:
Unless they actually are delivered as Ogg container streams?

Reply:
When asked to add an OGG file to the collection of radio stations, Transistor strips out the stream address and does not store the container.